files/auto/autopause-daemon.sh

```diff
@@ -81,17 +92,29 @@ do
     ;;
   XE)
     # Established
-    if ! java_clients_connected ; then
+    if -e /data/.skip-pause ; then
       TIME_THRESH=$(($(current_uptime)+$AUTOPAUSE_TIMEOUT_EST))
-      logAutopause "All clients disconnected - pausing in $AUTOPAUSE_TIMEOUT_EST seconds"
-      STATE=I
+      logAutopause "`/data/.skip-pause` file is present - skipping pausing"
+      STATE=E
+    else
+      if ! java_clients_connected ; then
+        TIME_THRESH=$(($(current_uptime)+$AUTOPAUSE_TIMEOUT_EST))
+        logAutopause "All clients disconnected - pausing in $AUTOPAUSE_TIMEOUT_EST seconds"
+        STATE=I
+      fi
```

Comment:
I'm not sure why this logic needs to change at all. The state transition still makes sense to me even if pause is being skipped.

Reply:
Alright, I'll revert the changes I made to the `Established` section.